cmake/TorchConfig.cmake.in

```diff
@@ -39,51 +60,95 @@ endif()
 if(@BUILD_SHARED_LIBS@)
   find_package(Caffe2 REQUIRED PATHS ${CMAKE_CURRENT_LIST_DIR}/../Caffe2)
   set(TORCH_LIBRARIES torch ${Caffe2_MAIN_LIBS})
+  find_library(C10_LIBRARY c10 PATHS "${TORCH_INSTALL_PREFIX}/lib")
+  list(APPEND TORCH_LIBRARIES ${C10_LIBRARY})
```

Comment:
Why not use `append_torchlib` macro here?

Reply:
I didn't plan to touch the code which belongs to shared library build. Anyway, I can fix this if it looks good.